rules_conditions.md

```diff
@@ -0,0 +1,226 @@
+Because YAML treats certain characters like less-than (`<`) specially, it is good practice to always enclose the basic comparison operators in single quotes (like `'<'`).
+
+The `Operator` may be one of the following:
+
+### `'='`
+
+Basic comparison -- `equals`.
+True if the value of the named Field is equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'!='`
+
+Basic comparison -- `not equals`.
+True if the value of the named Field is not equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'<'`
+
+Basic comparison -- `less than`.
+True if the value of the named Field is less than the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'<='`
+
+Basic comparison -- `less than or equal to`.
+True if the value of the named Field is less than or equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'>'`
+
+Basic comparison -- `greater than`.
+True if the value of the named Field is greater than the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### `'>='`
+
+Basic comparison -- `greater than or equal to`.
+True if the value of the named Field is greater than or equal to the `Value` specified.
+See [`Datatype`](#datatype) ffor how different datatypes are handled.
+
+### starts-with
+
+Tests if the span value named by the `Field` begins with the text specified in the `Value` parameter.
+Comparisons are case-sensitive and exact.
+
+Values are always coerced to strings -- the `Datatype` parameter is ignored.
+
+### contains
+
+Tests if the span value named by the `Field` contains the text specified in the `Value` parameter.
+Comparisons are case-sensitive and exact.
+
+Values are always coerced to strings -- the `Datatype` parameter is ignored.
+
+### does-not-contain
+
+Tests if the span value named by the `Field` does not contain the text specified in the `Value` parameter.
+Comparisons are case-sensitive and exact.
+
+Values are always coerced to strings -- the `Datatype` parameter is ignored.
+
+### exists
+
+Tests if the specified span contains the field named by the `Field` parameter, without considering its value.
+
+Both the `Value` and the `Datatype` parameters are ignored.
+
+### not-exists
+
+Tests if the specified span does not contain the field named by the `Field` parameter, without considering its value.
+
+Both the `Value` and the `Datatype` parameters are ignored.
+
+### matches
+
+Tests if the span value specified by the `Field` parameter matches the regular expression specified by the `Value` parameter.
+The regular expression grammar used is the syntax used by the Go programming language.
+It is documented [here](https://pkg.go.dev/regexp/syntax).
+
+For clarity, regular expressions in YAML should usually be quoted with single
```

Comment:
The formatting for these two paragraphs are different than the rest of the document. Were these sourced from somewhere else?

Reply:
I wrote them first when I was still writing the code. Forgot to fix them. Thanks for the poke.

Reply:
No worries, Kent. Thank you!